hknweb/events/templates/events/add_event.html

```diff
@@ -3,6 +3,7 @@
 {% block title %}Add an event{% endblock %}
 
 {% block header %}
+<link rel="stylesheet" href="{% static "events/style.css" %}">
 <style>
```

Comment:
Why would you move only part of the CSS into a style.css?

Reply:
Because some of it is specific to the html, I moved out as much as I could

Reply:
Because some of it is specific to the html, I moved out as much as I could